scipy/stats/morestats.py

```diff
@@ -2078,7 +2078,7 @@ def anderson_ksamp(samples, midrank=True):
 AnsariResult = namedtuple('AnsariResult', ('statistic', 'pvalue'))
 
 
-def ansari(x, y):
+def ansari(x, y, alternative='two-sided', mode='auto'):
```

Comment:
Oops, I used `method` in `page_trend_test`. KS tests and wilcoxon use `mode` - were there others?

Reply:
> KS tests and wilcoxon use `mode` - were there others?

Nope.

Reply:
I just found some discussion about this in gh-10796. Would you support changing KS tests and wilcoxon in favor of `mode`? We could deprecate `mode` and remove it from the documentation, but never actually remove support for it, so it can be done in a way that's backwards compatible.